drafts/WEP0002-slack-community-moderation-procedures.rst

```diff
@@ -0,0 +1,142 @@
+* A lack of tracking that could assist moderators in identifying whether a user's actions formed a pattern of negative behaviour (we mostly relied on memory instead).
+
+The "three strikes" model of escalating moderator actions attempts to identify both of these issues:
+
+* With three different levels of action, moderators have a framework of addressing even minor actions that may not constitute major violations of the Code of Conduct rules, but that may bother or alienate others over time.
+* A tracking spreadsheet allows moderators to look back on previous actions they've taken, even in the event of moderator turnover, and more easily identify when users are failing to respond to moderator intervention.
+
+When a user violates the Code of Conduct, moderators can engage in a series of escalating actions. Depending on the situation, these actions may not all be required: ideally, a user will not require further escalation beyond the first strike, and if the behaviour is particularly egregious, the moderation team may decide to jump forward to a more extreme action. Similarly, moderators can decide to use the same action twice rather than escalating if, for example, a user shows that they're making a good faith effort to follow the Code of Conduct. These steps are intended as tools for moderation and education, and have lots of room for nuance; they're not a one-way funnel intended to push users out of the community.
+
+Regardless of which action is taken, every time a moderator needs to take action, they should let the other moderators know and track it in the tracking spreadsheet. For privacy reasons, this spreadsheet should only be accessible to the moderation group (currently, the link to the spreadsheet is pinned in the #slack-mods channel, but we may want to change viewing/editing permissions to only people who are invited, rather than anyone who has the link). The spreadsheet can help the moderation team ensure that we're consistent in the types of conduct we respond to and our responses to it. We could consider an expiration date of, say, a year, at which point actions "expire" and won't be taken into account anymore - making an assumption that people learn over time, and won't have mistakes count against them forever.
+
+Depending on how visible the user's conduct was to the greater community, it may make sense to communicate moderation actions to the greater community (in the #announcements channel, or just in the channel the conversation(s) occurred in), both for transparency and to reassure community members that the moderation team is willing to act if they need us to.
+
+1. Verbal reminder
+__________________
+The first time a member of the moderation team contacts a user, or for constact over minor actions, it can be an in-channel reply or DM; at this point, we can assume that the user may not be cognisant of the expectations in the community.
```

Comment:
Not 100% sure this is the right edit. 

```suggestion
The first time a member of the moderation team contacts a user, or for multiple minor actions, it can be an in-channel reply or DM; at this point, we can assume that the user may not be cognisant of the expectations in the community.
```

Reply:
Whoops, thanks for catching that typo! I meant "contact," which could just happen once.